app/views/admin/administrators/administrators/delete.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Confirm deletion of <%= @administrator.full_name %></h1>
```

Comment:
Can you show this screen to Al and Shani for a design review?

Reply:
Checked and done